scripts/uninstall.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+# This script uninstalls Ollama.
+
+set -eu
+
+# default error condition
```

Comment:
The logic is your script is good but the comments seem very verbose. I'd recommend sticking with the style of the original install.sh script and remove all the comments except for one or two where you think the code is not already self-explantory (or refactor the code so it doesn't need a comment).

The function names are already good and the `echo` statements also tell us what is going on in their message, so those comments are not needed then, for example.